### DISCUSSION TITLE
OpenRCX query

### DISCUSSION THREAD

**Discussion by faisaladilquadery** [The-OpenROAD-Project/OpenROAD#4287] [General]
I just want to know if OpenRCX a engine integrated in OpenRoad or is it a RC signoff tool  like Cadence Quantas

**Comment by maliberty** [comment]
It is integrated with OR but aspires to be signoff quality.  During optimization we use estimated parasitics.

**Comment by faisaladilquadery** [comment]
thankyou @maliberty
